ragengine/vector_store/faiss_store.py

```diff
@@ -0,0 +1,228 @@
+import os
+from typing import Dict, List
+
+import faiss
+from llama_index.core import Document as LlamaDocument
+from llama_index.core import (StorageContext, VectorStoreIndex, load_index_from_storage)
+from llama_index.core.storage.index_store import SimpleIndexStore
+from llama_index.core.storage.docstore.types import RefDocInfo
+from llama_index.vector_stores.faiss import FaissVectorStore
+
+from models import Document
+from inference.custom_inference import CustomInference
+
+from config import PERSIST_DIR
```

Comment:
if import pkg in this way, can python find the correct local pkg? not some python built-in library.
I think the correct way may be `from ragengine.config import XXXX`

Reply:
right, I was editing the syspath in conftest.py using absolute paths is better, done